debug/pylint.rc

```diff
@@ -286,4 +286,4 @@ max-public-methods=20
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "Exception"
-overgeneral-exceptions=Exception
+#overgeneral-exceptions=Exception
```

Comment:
Latest pylint complains about it:
```
pylint: Command line or configuration file:1: UserWarning: Specifying exception names in the overgeneral-exceptions option without module name is deprecated and support for it will be removed in pylint 3.0. Use fully qualified name (maybe 'builtins.Exception' ?) instead.
```

Since we're just using the default value I commented it out.